src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -71,61 +72,82 @@ def fetch_group_to_event_data(
     return buffer.backend.get_hash(model=model, field=field)
 
 
-def get_dcg_group_workflow_data(
-    worklow_event_dcg_data: dict[str, str]
```

Comment:
lulz - this is why we need colleen to review our code 🤣 (she always catches this kind of thing for me 🙏)